boxoffice/extapi/razorpay.py

```diff
@@ -0,0 +1,19 @@
+import requests
+from boxoffice import app
+
+
+base_url = 'https://api.razorpay.com/v1/payments'
+
+__all__ = ['capture_payment']
+
+
+def capture_payment(payment_id, amount):
+    """
+    Attempts to capture the payment, from Razorpay
+    """
+    verify_https = False if app.config.get('VERIFY_RAZORPAY_HTTPS') is False else True
+    url = '{base_url}/{payment_id}/capture'.format(base_url=base_url, payment_id=payment_id)
```

Comment:
This will break if the `base_url` above has a trailing slash. Leave a comment there warning about not having a trailing slash, or use urljoin.